docs/JLBP-0006.md

```diff
@@ -93,7 +93,7 @@ Given this scenario, here are the possible combinations of renamings:
 - **Rename Java package**:
   - **Case 3: Keep Maven ID**. The classes from `g1:a1:1.0.0` and
     `g1:a1:2.0.0` could technically be used together, but since they share the
-    Maven group ID and artifact ID, only one jar can be pulled in.
+    Maven group ID and artifact ID, only one artifact can be pulled in.
```

Comment:
I think JAR file is right here. "pulled in" really means (and probably should be rewritten as) "added to the class path". JAR files are added to the class path. Artifacts aren't.  What do you think?

Reply:
How about "_only the classes in one of the artifacts can be pulled in_"? This sentence starts with "The classes" and classes are part of class path (through JAR files).

Reply:
Updated to "the build system selects only one of them" (active voice; without using "pull in")